src/Nethermind/Nethermind.Facade/Filters/FilterStore.cs

```diff
@@ -56,6 +56,10 @@ public FilterType GetFilterType(int filterId)
         public IEnumerable<T> GetFilters<T>() where T : FilterBase =>
             _filters.Select(f => f.Value).OfType<T>();
 
+        public T GetFilter<T>(int filterId) where T : FilterBase => _filters.TryGetValue(filterId, out var filter)
+                ? filter as T
+                : throw new InvalidOperationException($"Filter with ID {filterId} not found");
```

Comment:
Wouldn't nullable `T?` be better?

Reply:
Corrected to avoid throwing an exception.